src/js/components/Overview/PublicOverview.tsx

```diff
@@ -59,6 +60,8 @@ const PublicOverview = () => {
     saveToLocalStorage(sections);
   }, [sections]);
 
+  if (!selectedProject) return <Catalogue />;
```

Comment:
does this work with single-project nodes? I know there's an auto-select, but does the catalog flicker in and out when the node only has one project? should it be loader-aware since below this early return skips the loading...